pshchegolevatykh/CountingBitsNaive.cs

```diff
@@ -0,0 +1,20 @@
+public class Solution {
+    public int[] CountBits(int n) {
+        var bits = new int[n+1];
+
+        for (int i = 0; i < n+1; i++) {
+            var currentDecimal = i;
+            int bitsCount = 0;
+            while (currentDecimal > 0) {
+                int remainder = currentDecimal % 2;

```

Comment:
So cool, we can just count the non-null remainders! First I thought of this too, but then fell into doubt and just used Math.Log2

Reply:
👍